OCPBUGS#12622: Fixed Statically provisioning hostPath volumes bug (#84664)
openshift/openshift-docs: modules/persistent-storage-hostpath-static-provisioning.adoc

.Procedure

. Define the persistent volume (PV) by creating a `pv.yaml` file with the `PersistentVolume` object definition:
+
[source,yaml]
----
apiVersion: v1
kind: PersistentVolume
metadata:
  name: task-pv-volume <1>
  labels:
    type: local
spec:
  storageClassName: manual <2>
  capacity:
    storage: 5Gi
  accessModes:
    - ReadWriteOnce <3>
  persistentVolumeReclaimPolicy: Retain
  hostPath:
    path: "/mnt/data" <4>
----
<1> The name of the volume. This name is how the volume is identified by persistent volume claims or pods.

> <2> Used to bind persistent volume claim requests to this persistent volume.

Reviewer: ```suggestion
<2> Used to bind persistent volume claim requests to this PV.
```
Since you've already used "PV" to mean "persistent volume" in this section, consider using "PV" here.

Author: :heavy_check_mark:

<3> The volume can be mounted as `read-write` by a single node.

> <4> The configuration file specifies that the volume is at `/mnt/data` on the cluster's node. Do not mount to the container root, `/`, or any path that is the same in the host and the container. This can corrupt your host system. It is safe to mount the host by using `/host`.

Reviewer: ```suggestion
<4> The configuration file specifies that the volume is at `/mnt/data` on the cluster's node. To avoid corrupting your host system, do not mount to the container root, `/`, or any path that is the same in the host and the container. It is safe to mount the host by using `/host`.
```
I tried to revise this section a bit to avoid starting a sentence with an ambiguous "This", per IBM Style. See https://www.ibm.com/docs/en/ibm-style?topic=grammar-pronouns#ambiguous-pronoun-references.

Author: :heavy_check_mark:
